### PR TITLE
Add migration documentation

### PR DESCRIPTION
**Pull request by hickeyma**
This document describes how to migrate from Helm v2 to v3.

### CODE REVIEW

**Comment by rimusz** [comment]
LGTM

File: content/docs/intro/v2_v3_migration.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+4. XDG directory specification added:
+   - Helm home removed and replaced with XDG directory specification for storing configuration files
+   - No longer need to initialize Helm
+   - `helm init` and `helm home` removed
+5. Additional changes:
+   - Helm install/set-up is simplified:
+     - Helm client (helm) only (no tiller)
+     - Run-as-is paradigm
+   - `local` or `stable` repositories are not set-up by default
+   - `crd-install` hook removed and replaced with `crd` directory in chart where all CRDs defined in it will be installed before any rendering of the chart
```
Comment (thomastaylor312) [inline content/docs/intro/v2_v3_migration.md]: The directory is actually `crds`

Comment (hickeyma) [reply content/docs/intro/v2_v3_migration.md]: Good catch. Updated.

**Comment by hickeyma** [comment]
Thanks for the review @thomastaylor312. Comment updated now and ready for review again.

**Comment by rimusz** [comment]
LGTM

**Pull request merged**